client/src/pages/boxes/SettleBoxPage4/SettleBoxPage4.tsx

```diff
@@ -0,0 +1,186 @@
+import { Item } from 'rc-menu';
+import s from '../BoxesPage.module.less';
+import { Button } from 'antd';
+
+interface BoxData {
+  volunteerId: string;
+  boxId: string;
+  plnAmount: Record<string, number>;
+  foreignCurrency: Record<string, number>;
+  other?: string;
+}
+
+export const SettleBoxPage4 = () => {
```

Comment:
I'd suggest more descriptive names. E.g. SettleBoxPageCheckout. These numbers will seem magical in a matter of days